examples/with-context-api/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "with-context-api",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "latest",
```

Comment:
Let's use canary here, as it doesn't work on 6.x

Reply:
That should be `"next": "7.0.0-canary.16"` ?

Sorry, my first PR 😄

Reply:
"next": "^7.0.0-canary.16",

Reply:
You should be good with `yarn add next@canary` :+1: